eng/outerbuild/outerBuild.targets

```diff
@@ -0,0 +1,22 @@
+<Project>
+  <Target Name="GenerateReferenceSource" DependsOnTargets="RunOuterBuildForProjects">
+    <MSBuild Projects="@(InnerBuildProjects)"
+             Targets="GenerateReferenceSource"
+             BuildInParallel="$(BuildInParallel)">
+    </MSBuild>
+  </Target>
+
+  <Target Name="RunOuterBuildForProjects">
```

Comment:
Also just to make sure, mi initial comment was if we were already doing this somewhere else? I mean, Choosing the best target framework and then adding the target frameworks to projects? Because running these two steps seem pretty common.

If we do this in various places, why not put this in a central target somewhere in the configuration system SDK and depend on that target? Do you get what I mean?

Reply:
Right, we should be sharing a common target for this.  Not creating a new one.